futaba/cogs/settings/core.py

```diff
@@ -342,3 +343,55 @@ def mention(role):
 
         await ctx.send(embed=embed)
         self.journal.send("roles/jail", ctx.guild, content, icon="settings", role=role)
+
+    @commands.command(name="trackblacklistadd")
+    @commands.guild_only()
+    @permissions.check_mod()
+    async def tracker_blacklist_add(
+        self, ctx, *, user_or_channel: Union[MemberConv, TextChannelConv]
+    ):
+        """ Add a user or channel to the tracking blacklist. """
+
+        logger.info(
+            "Adding '%s' (%d) to the tracking blacklist for guild '%s' (%d)",
```

Comment:
Maybe add a `"user" if isinstance(user_or_channel, discord.Member) else "channel"` to the log.
E.g. "Adding user 'aismallard' (111111111111) to the tracking blacklist for guild ..."